core/mm/vm.c

```diff
@@ -799,10 +799,14 @@ static TEE_Result map_kinit(struct user_mode_ctx *uctx)
 	size_t offs;
 	vaddr_t va;
 	size_t sz;
+	uint32_t prot;
```

Comment:
Please initialize all locals.

Reply:
Since the other locals were not initialised I left prot also the same. My bad. I will initialise prot here and create a separate commit for initialising the other local variables.

Reply:
Doing it in the same commit is also fine.